docs/style.md

```diff
@@ -0,0 +1,50 @@
+---
+id: style
+title: Styleguides
+sidebar_label: Styleguides
+---
+
+---
+
+
+On `amphora-html` using the `data`  and `meta` object that are the instances of the components and site respectively. This state
```

Comment:
Theres a double space after `data`
```suggestion
On `amphora-html` using the `data` and `meta` object that are the instances of the components and site respectively. This site
```
My guess is that you wanted to say `This site` instead of `This state`

Reply:
Done!!